### PR TITLE
Update UG according to Jane's comment

### PR DESCRIPTION
**Pull request by hhdqirui**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -870,21 +881,22 @@ Examples:
...
+a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 ### Exiting the program
 
-Exits the program.
+If you want to exits the program, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:899]: A small typo here, if you want to exit

File: docs/UserGuide.md [hunk 2]
```diff
@@ -870,21 +881,22 @@ Examples:
...
 
-Clears all entries from VirusTracker.
+If you want to clear all entries from VirusTracker, you can use the following command.
 
 Format: `clear`
 
 ### Viewing help
 
-Shows a message explaining how to access the help page.
+If you are sure how to use VirusTracker and want get some help, you can use the following command which will show
```
Comment (DreamerDragon) [inline docs/UserGuide.md:890]: If you are unsure of how to use

Comment (DreamerDragon) [reply docs/UserGuide.md:890]: maybe can say how to access the user guide for the instructions?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -801,7 +812,7 @@ Examples:
 
 ### Finding persons by name
 
-Finds persons whose names contain any of the given keywords.
+It you want to find persons whose names contain any of the given keywords, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:815]: Small typo here, If you

File: docs/UserGuide.md [hunk 4]
```diff
@@ -400,34 +400,44 @@ Examples:
...
-* Updates the visits list to displays all visits currently stored in VirusTracker.
+##### What it does
+* It updates the visits list to displays all visits currently stored in VirusTracker.
 
 <div style="page-break-after: always;"></div>
 
 #### Listing high risk locations
-
+If you want to view a list of high risk locations, you can use the following command.
 Lists the locations with high risk of Covid infection.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:441]: Maybe remove this line, if it is already included in the line above?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -400,34 +400,44 @@ Examples:
...
 #### Listing all locations
+It you want ot view information of all locations stored in VirusTracker, you can use the following command.
 Format: `list l/locations`
 
-* Updates the locations list to displays all locations currently stored in VirusTracker.
+##### What it does
+* It updates the locations list to displays all locations currently stored in VirusTracker.
 
 #### Listing all visits
+It you want to view information of all visis stored in VirusTracker, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:431]: small typo here, if

File: docs/UserGuide.md [hunk 6]
```diff
@@ -400,34 +400,44 @@ Examples:
...
 #### Listing all quarantined people 
+It you want to view a list of all quarantined people, you can use the following command.
 Format: `list l/quarantined`
 
-* Filters the persons list to display all people currently in quarantine.
+##### What it does
+* It filters the persons list to display all people currently in quarantine.
 
 #### Listing all locations
+It you want ot view information of all locations stored in VirusTracker, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:424]: small typo here, if

File: docs/UserGuide.md [hunk 7]
```diff
@@ -400,34 +400,44 @@ Examples:
...
 #### Listing all infected people 
+If you want to view a list of all infected people, you can use the following command.
 Format: `list l/infected`
 
-* Filters the persons list to display all people that are currently infected.
+##### What it does
+* It filters the persons list to display all people that are currently infected.
 
 #### Listing all quarantined people 
+It you want to view a list of all quarantined people, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:417]: small typo here, if

File: docs/UserGuide.md [hunk 8]
```diff
@@ -373,7 +373,7 @@ Examples:
 
 #### Adding a visit
 
-Adds a visit by the person, location of visit and date of visit
+When you want to add a visit, you can use the following command to add a visit by the person, location of visit and date of visit
```
Comment (DreamerDragon) [inline docs/UserGuide.md:376]: maybe change to if? since the below part all used if, to make it look more consistent.

File: docs/UserGuide.md [hunk 9]
```diff
@@ -355,7 +355,7 @@ Examples:
 
 #### Adding a location
 
-Adds a location to VirusTracker.
+When you want to add a location to VirusTracker, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:358]: same as below

File: docs/UserGuide.md [hunk 10]
```diff
@@ -330,11 +330,11 @@ _Please take note of the above when using indexes._
 
 ### Adding data
 
-To add data to VirusTracker, there are `add` commands for each entity.
+If you want to add data to VirusTracker, there are `add` commands for each entity.
 
 #### Adding a person
 
-Adds a person to VirusTracker.
+When you want to add a person to VirusTracker, you can use the following command.
```
Comment (DreamerDragon) [inline docs/UserGuide.md:337]: same as below

**Review by DreamerDragon — CHANGES_REQUESTED** [review]
Looks good after change, but there are some typo to be fixed.

**Review by hopinxian — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (hhdqirui) [reply docs/UserGuide.md:899]: Fixed.

File: docs/UserGuide.md (hunk 2 above)
Comment (hhdqirui) [reply docs/UserGuide.md:890]: Fixed.

File: docs/UserGuide.md (hunk 3 above)
Comment (hhdqirui) [reply docs/UserGuide.md:815]: Fixed.

File: docs/UserGuide.md (hunk 4 above)
Comment (hhdqirui) [reply docs/UserGuide.md:441]: Fixed.

File: docs/UserGuide.md (hunk 5 above)
Comment (hhdqirui) [reply docs/UserGuide.md:431]: Fixed.

File: docs/UserGuide.md (hunk 6 above)
Comment (hhdqirui) [reply docs/UserGuide.md:424]: Fixed.

File: docs/UserGuide.md (hunk 7 above)
Comment (hhdqirui) [reply docs/UserGuide.md:417]: Fixed.

File: docs/UserGuide.md (hunk 8 above)
Comment (hhdqirui) [reply docs/UserGuide.md:376]: Fixed.

File: docs/UserGuide.md (hunk 9 above)
Comment (hhdqirui) [reply docs/UserGuide.md:358]: Fixed.

File: docs/UserGuide.md (hunk 10 above)
Comment (hhdqirui) [reply docs/UserGuide.md:337]: Fixed.

**Review by DreamerDragon — APPROVED** [review]

**Pull request merged**